pkg/api/v1alpha1/conversion.go

```diff
@@ -19,7 +19,6 @@ package v1alpha1
 import (
 	"context"
 	"fmt"
-
```

Comment:
Can you please undone the new line removal? To keep this file unchanged.

Reply:
Done in https://github.com/kubernetes-sigs/descheduler/pull/1475/commits/61b7b995b99122065a13a0fa8ff87fbccbc91d52